tests/test_files/ca.pem

```diff
@@ -0,0 +1,50 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
There is nothing sensitive in these certificates is there.  I assume they were generated very specifically for the test cases and are not used anywhere else?

Reply:
Certificates should be roughly the equivalent of public keys, so there's nothing sensitive. Private keys might be. Given that the password is "test" to decrypt the private keys, I'm pretty sure these are the same as the ones used in the Asio SSL example: https://github.com/chriskohlhoff/asio/tree/master/asio/src/examples/cpp11/ssl